src/sentry/static/sentry/app/views/projectDetail/charts/sessionsRequest.tsx

```diff
@@ -282,6 +302,32 @@ class SessionsRequest extends React.Component<Props, State> {
     };
   }
 
+  transformSessionCountData(
+    responseData: SessionApiResponse
+  ): {
```

Comment:
do we need the output type?

Reply:
it helped me during development, but I can remove it now 👍